build.gradle

```diff
@@ -28,7 +28,7 @@ targetCompatibility = 1.8
 // To test with snapshots, modify the version directly in the dependencies section,
 // but don't forget to change it back when you're done.
 ext.versions = [
-	'dotifyApi'		: '5.0.0',
+	'dotifyApi'		: '5.0.1-SNAPSHOT',
```

Comment:
dotify.api 5.0.1 is now released so please update this so the tests might pass.

Reply:
Hmm, I just realized you might want to increase the major version because this was a backward incompatible change.